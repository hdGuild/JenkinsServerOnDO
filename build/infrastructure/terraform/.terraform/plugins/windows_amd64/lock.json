{
  "digitalocean": "4149811bcb07e7e29a1f95744648dc34540f206b7f0f70dc5b18fdc9204f6db2",
  "local": "88d2c87f981ebfcac072b361f09fd22c3b387958df0df4790328177c1d23ad67"
}