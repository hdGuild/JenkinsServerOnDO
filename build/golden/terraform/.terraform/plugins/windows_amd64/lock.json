{
  "digitalocean": "4149811bcb07e7e29a1f95744648dc34540f206b7f0f70dc5b18fdc9204f6db2"
}